history/characters/29000_gnoll.txt

```diff
@@ -1602,12 +1602,12 @@
 	584.10.25={ death=yes }
 }
 29220={
-	name=Gorp
+	name=Snarlmane
```

Comment:
Why you swapped their names?

Reply:
There is already another Snarlmane[29025], landed in c_winfried by 603.10.21. I changed his name to Ripsnarl.
The reason I selected [29220] to be Sylvanas' Snarlmane is because his existing character history is what matches the closest to the lore described on WoWpedia: he becomes scourged and Necromancer Thule's [36052] friend on Fenris Isle in 605.5.5.

Alternately I can set Ripsnarl (formerly also Snarlmane) of c_winfried as a vassal to Sylvanas but it introduces balance concerns since this PR would be adding 2 vassals to Sylvanas. I would also have to transplant [29220]'s history with becoming scourged and Thule's friend over to [29025], and create a new holder for c_winfried.

Reply:
> Alternately I can set Ripsnarl (formerly also Snarlmane) of c_winfried as a vassal to Sylvanas but it introduces balance concerns since this PR would be adding 2 vassals to Sylvanas. I would also have to transplant [29220]'s history with becoming scourged and Thule's friend over to [29025], and create a new holder for c_winfried.

I would keep this Ripsnarl independend and pick another county somewhere in Tirisfal for new Snarlmane: maybe Riverborn?